lesson13/index.js

```diff
@@ -0,0 +1,132 @@
+const contents = document.getElementById('js-contents');
+
+window.addEventListener('DOMContentLoaded', () => {
+    // モーダルを表示させるボタンを表示する
+    showModalOpenButton();
+});
```

Comment:
Q.
これ使う理由はなんですか？

Reply:
Since I'm loading the script file just before `</ body>`, the code will be executed after the DOM tree is built, even if I don't write DOMContentLoaded.
However, I wrote it to make it clear that it will fire after the DOM tree has been built.
I think it's okay to write it if it is the same whether it is written or not.